sdk/quantum/tests.yml

```diff
@@ -0,0 +1,16 @@
+trigger: none
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      ServiceDirectory: quantum
+      Artifacts:
+        - name: azure-quantum-jobs
+          groupId: com.azure.quantum.jobs
+          safeName: azurequantumjobs
+      EnvVars:
+        AZURE_TEST_MODE: PLAYBACK
```

Comment:
This yml generally is used to configure live testing against deployed service resources, `AZURE_TEST_MODE: PLAYBACK` shouldn't run against deployed resources and instead should be used to run against recorded tests. This should be changed to either `LIVE` or `RECORD`